starfish/data/__init__.py

```diff
@@ -53,7 +53,9 @@ def osmFISH(use_test_data: bool=False):
     """
     if use_test_data:
         return Experiment.from_json(
-            'https://d2nhj9g34unfro.cloudfront.net/20181005/osmFISH/experiment.json')
+            'https://d2nhj9g34unfro.cloudfront.net/browse/formatted/20181217/osmFISH/'
```

Comment:
```suggestion
            "https://d2nhj9g34unfro.cloudfront.net/browse/formatted/20181217/osmFISH/"
```

Reply:
I changed all of `starfish.data`, which was using `'`.